.github/workflows/build_and_test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Upload Linux Client
         uses: actions/upload-artifact@v2
         with:
-          name: unsigned-linux-client
+          name: debug-linux-client
```

Comment:
Perhaps we name it `client-[platform]-[mode]` so we can get better grouping in sorted order. (see all the stuff for the same platform together).

Reply:
yes, that'll play well down the line